appendices/glossary.txt

```diff
@@ -711,6 +716,18 @@ Image::
     of device memory.
     Represented by a slink:VkImage object.
 
+ifdef::VK_EXT_present_timing,VK_GOOGLE_display_timing[]
+Image Present Duration::
+    The amount of time the application intends for each
+    newly-presented image to be visible to the user.
+    This value should: be a multiple of the refresh cycle duration.
```

Comment:
This language may be valid for other presentation modes, but not for immediate present mode.